pkg/rancher-desktop/integrations/windowsIntegrationManager.ts

```diff
@@ -206,6 +206,7 @@ export default class WindowsIntegrationManager implements IntegrationManager {
         this.syncDistroSocketProxy(distro, state),
         this.syncDistroDockerPlugins(distro, state),
         this.syncDistroKubeconfig(distro, kubeconfigPath, state),
+        this.syncDistroSpinCLI(distro, state),
```

Comment:
We're running this without checking if WASM is enabled?

Reply:
We use the kubernetes setting as part of the input there:
https://github.com/rancher-sandbox/rancher-desktop/blob/d17f97f35918f60716a5380634c9faa0c185bb26/pkg/rancher-desktop/integrations/windowsIntegrationManager.ts#L484